tests/test_url_creator.py

```diff
@@ -48,73 +53,46 @@ def skip(platform, repo, arch):
     return False
 
 
-def test_configuration(repo, platform, arch=None, buildtype=None):
-    print("Testing download for ({},{},{},{})".format(repo, platform, arch, buildtype))
-    assert platform in PLATFORMS
+def test_configuration(repo, platform, arch=None, buildtype=None, cset='latest'):
```

Comment:
It seems `cset` is only used in printing, but not in the `url_creator` call. Should it be?